packages/api/src/controllers/generate.test.ts

```diff
@@ -237,6 +243,19 @@ describe("controllers/generate", () => {
       });
       expect(aiGatewayCalls).toEqual({ "segment-anything-2": 1 });
     });
+
+    it("should call the AI Gateway for generate API /llm", async () => {
+      const res = await client.fetch("/beta/generate/llm", {
+        method: "POST",
+        body: buildForm({ prompt: "foo" }),
```

Comment:
lol so the LLM API is a multipart form even tho there's no files in the payload? or is it multi-modal and _could_ receive files? if it could, I'd say we test with a dummy file here as well just in case

Reply:
yeah.... it did strike me as odd too but wasn't sure. there aren't any file fields in the spec 😬